src/RealMeService.php

```diff
@@ -730,7 +730,7 @@ public function getCertificateContents($certPath, $contentType = 'certificate')
                 if ($contentType == 'certificate') {
                     $pattern = '/-----BEGIN CERTIFICATE-----[\r\n]*([^-]*)[\r\n]*-----END CERTIFICATE-----/';
                 } elseif ($contentType == 'key') {
-                    $pattern = '/-----BEGIN [A-Z ]*PRIVATE KEY-----\n([^-]*)\n-----END [A-Z ]*PRIVATE KEY-----/';
+                    $pattern = '/-----BEGIN [A-Z ]*PRIVATE KEY-----[\r\n]*([^-]*)[\r\n]*-----END [A-Z ]*PRIVATE KEY-----/';
```

Comment:
The linter doesn't like this line either - it's a tiny bit too long.

Reply:
to get into the 120 character limit, we could cut some characters with something like this:

```
-{5}BEGIN [A-Z ]*PRIVATE KEY-{5}[\r\n]*([^-]*)[\r\n]*-{5}END [A-Z ]*PRIVATE KEY-{5}
```

but imo that's a bit ugly. or we could split up the regex like this:

```php
                    $pattern = '/-----BEGIN [A-Z ]*PRIVATE KEY-----[\r\n]*([^-]*)[\r\n]*'
                        . '-----END [A-Z ]*PRIVATE KEY-----/';
```

what one would you prefeer @ScopeyNZ

Reply:
🤷‍♂ I'm indifferent. The second one is what I was expecting - I agree the first is a little more complex.

Reply:
sweet, @ScopeyNZ i've updated it with the two lined approach